network.go

```diff
@@ -74,13 +81,20 @@ type Network interface {
 	// Labels support will be added in the near future.
 	CreateEndpoint(name string, sboxKey string, options interface{}) (Endpoint, error)
 
-	// Endpoints returns the list of Endpoint in this network.
+	// Endpoints returns the list of Endpoint(s) in this network.
 	Endpoints() []Endpoint
 
+	// WalkEndpoints uses the provided function to walk the Endpoints
+	WalkEndpoints(walker EpWalker)
+
 	// Delete the network.
 	Delete() error
 }
 
+// NwWalker is a client provided function which will be used to walk the Networks.
+// When the function returns true, the walk will stop
+type NwWalker func(nw Network) bool
```

Comment:
Being an API exposed to the applications, shorter forms of names such as Nw and Ep as part of the Type doesn't feel right. This is just a preference. I will check with @dave-tucker to see what he thinks about this.